Portal/src/Datahub.Portal/i18n/localization.fr.json

```diff
@@ -1688,13 +1688,16 @@
       "SharingAPIs": "Les API de partage de DataHub",
       "Metadata": "Gestion des métadonnées",
       "MonthlyCost": "Coûts mensuels",
+      "WhitelistProjectResources": "Ressources du projet de liste blanche"
+
     },
     "CARD_DESCRIPTIONS": {
       "ExternalSharing": "Gérer le partage externe des fichiers et des rapports Power BI",
       "ManageServices": "Approuver la création de services et les demandes d'accès",
       "Registrations": "Gérer les enregistrements",
       "SharingAPIs": "Gérer les clés API pour les utilisateurs externes",
       "Metadata": "Gérer le modèle de métadonnées de Datahub",
+      "WhitelistProjectResources": "Ressources du projet de liste blanche"
```

Comment:
Gérer les ressources auxquelles l'espace de travail a accès

Reply:
thanks, fixed in commit 374ce84024c2adfa607377231094e98b9ee48b1b